src/redux/reducer.js

```diff
@@ -63,6 +65,8 @@ const initialState = {
     selectedFile: null,
     uploadingElements: {},
     directoryUpdated: { force: 0, eventData: {} },
+    treeMapData: {},
```

Comment:
Well we could have done only one state in redux store, something like : 
`treeData: { mapData: {}, rootDirectories: []},`